src/main/java/by/dragonsurvivalteam/dragonsurvival/client/render/ClientDragonRender.java

```diff
@@ -491,16 +552,18 @@ public static void renderEntityInInventory(LivingEntity entity, int x, int y, fl
 			if(ClientDragonRender.dragonArmor == null){
 				ClientDragonRender.dragonArmor = DSEntities.DRAGON_ARMOR.create(Minecraft.getInstance().player.level);
 				assert dragonArmor != null;
-				ClientDragonRender.dragonArmor.player = Minecraft.getInstance().player.getId();
+				ClientDragonRender.dragonArmor.playerId = Minecraft.getInstance().player.getId();
 			}
 
 			if(!ClientDragonRender.playerDragonHashMap.containsKey(Minecraft.getInstance().player.getId())){
 				DragonEntity dummyDragon = DSEntities.DRAGON.create(Minecraft.getInstance().player.level);
-				dummyDragon.player = Minecraft.getInstance().player.getId();
+				dummyDragon.playerId = Minecraft.getInstance().player.getId();
 				ClientDragonRender.playerDragonHashMap.put(Minecraft.getInstance().player.getId(), new AtomicReference<>(dummyDragon));
 			}
 		}
+		// FIXME :: "Could not load animation: sitting_blep. Is it missing?"
```

Comment:
may have been renamed in the last update. If there are any "non-existent" emotions left there, you can delete them. The ones that are there now are the actual ones.